libs/fbthrift/Makefile

```diff
@@ -1,49 +1,56 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=fbthrift
-PKG_VERSION:=2019.05.27.00
+PKG_VERSION:=2019.06.10.00
 PKG_RELEASE:=1
-PKG_MAINTAINER:=Amol Bhave <ambhave@fb.com>
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/facebook/fbthrift/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=7c56dae532e60844520cef4ce03abd4911323fe340fdee0cc3a6a3e8adb09e3f
+PKG_HASH:=0897c66564a2742b24dd6ae77812b42987c3493944d1d5d40d4005d71deb7cb5
 
+PKG_MAINTAINER:=Amol Bhave <ambhave@fb.com>
+PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DEPENDS:=fbthrift/host
 HOST_BUILD_DEPENDS:=boost/host libmstch/host
-
-include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/cmake.mk
+PKG_BUILD_DEPENDS:=fbthrift/host
 
 HOST_BUILD_PREFIX:=$(STAGING_DIR_HOST)
-
-PKG_BUILD_PARALLEL:=1
 HOST_BUILD_PARALLEL:=1
-CMAKE_OPTIONS:= \
-	-DBoost_NO_BOOST_CMAKE=ON \
-	-DCMAKE_POSITION_INDEPENDENT_CODE=ON \
-	-DBUILD_SHARED_LIBS=ON
-CMAKE_HOST_OPTIONS:= \
-	-DBoost_NO_BOOST_CMAKE=ON \
-	-Dcompiler_only=ON
-HOST_LDFLAGS:=-Wl,-rpath=$(STAGING_DIR_HOST)/lib
+PKG_BUILD_PARALLEL:=1
 CMAKE_INSTALL:=1
 
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/cmake.mk
+
 define Package/fbthrift
 	SECTION:=libs
 	CATEGORY:=Libraries
 	TITLE:=Facebook's branch of Apache Thrift, including a new C++ server.
-	DEPENDS:=+libwangle +libfmt +libmstch +librsocket-cpp +@boost-host-build-filesystem \
-		+@boost-host-build-thread +@boost-host-build-system
+	DEPENDS:=+libwangle +libfmt +libmstch +librsocket-cpp \
+		+@boost-host-build-chrono +@boost-host-build-date_time +@boost-host-build-atomic \
+		+@boost-host-build-thread +@boost-host-build-system +@boost-host-build-filesystem
 endef
 
 define Package/fbthrift/description
 	Facebook's branch of Apache Thrift, including a new C++ server.
 endef
 
+CMAKE_HOST_OPTIONS += \
+	-DBoost_NO_BOOST_CMAKE=ON \
+	-Dcompiler_only=ON
+
+HOST_LDFLAGS += -Wl,-rpath=$(STAGING_DIR_HOST)/lib
+
+CMAKE_OPTIONS += \
+	-DBoost_NO_BOOST_CMAKE=ON \
+	-DBoost_INCLUDE_DIRS="$(STAGING_DIR_HOST)/include/boost" \
+	-DBoost_LIBRARY_DIRS="$(STAGING_DIR_HOST)/lib" \
```

Comment:
The previous code does not have the three lines which include `Boost_INCLUDE_DIRS, Boost_LIBRARY_DIRS, and THRFIT1`.
The paths here are also incorrect since they are including headers and libs from the host build, where as the actual cmake build would need the target boost build.

Reply:
I added these as I get a different failure locally (cannot find libboost_filesystem.so) even though they are present. Still does not work with these.